integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java

```diff
@@ -301,10 +308,27 @@ private void kinit(SdxTestDto sdxTestDto, SdxClient sdxClient) {
         }
     }
 
+    private void dnsLookups(SdxTestDto sdxTestDto, SdxClient sdxClient) {
+        InstanceMetaDataV4Response instanceGroupMetadata = getInstanceGroups(sdxTestDto, sdxClient).stream()
+                .flatMap(instanceGroup -> instanceGroup.getMetadata().stream())
+                .filter(metadata -> metadata.getInstanceGroup().equals("idbroker"))
+                .filter(metadata -> StringUtils.isNoneBlank(metadata.getDiscoveryFQDN(), metadata.getPrivateIp()))
+                .findFirst().orElseThrow(() -> new TestFailException("FreeIPA upgrade DNS lookups test failed, idbroker instance group was not found."));
+        try {
+            String cmd = String.format(CHECK_DNS_LOOKUPS_CMD, instanceGroupMetadata.getDiscoveryFQDN(), instanceGroupMetadata.getPrivateIp());
+            Map<String, Pair<Integer, String>> results = sshJClientActions.executeSshCommand(getInstanceGroups(sdxTestDto, sdxClient),
+                    List.of(HostGroupType.MASTER.getName()), cmd, false);
+            results.values().forEach(result -> Assertions.assertTrue(result.getLeft() == 0));
```

Comment:
for equality you should use `assertEquals`

Reply:
Done.